### ISSUE TITLE
All images load at the same time

### ISSUE THREAD

**Issue by Jonotespere** [eisbehr-/jquery.lazy#142]
Hi, first of all congrats for this script!
I don't know why but all the images seem to load at the same time on my test page. There are no errors in the console, but images load at the same time, not as i scroll the page.
The script although seems to work partially, 'cause image URLs are moved by the script from the "data-src" attribute to "src", and an inline CSS style is added (display: inline-block).
It surely is a problem on my side: is there a way to understand if the script is working correctly?
Thanks!

**Comment by eisbehr-** [comment]
Hello @Jonotespere 

Do you have an `DOCTYPE` set in your HTML, and are your image elements has a default width and height set? This are the most common issues.

**Comment by Jonotespere** [comment]
Yes, i do have a doctype, so do i have to remove it (like in your test
page)?
No, width and height are not set in this test page.
Thank you! I try these two advices,

Bye!
Mattia

Il giorno mer 4 apr 2018 alle 12:53 Eisbehr <notifications@github.com> ha
scritto:

**Comment by eisbehr-** [comment]
No! You **need** a `DOCTYPE`! (and my page has one too?!)

And you need a default width and height too. Otherwise the browser will think all your images as a size of zero by zero pixels. Then all your images will fit into the page at once and Lazy will load all.

**Comment by Jonotespere** [comment]
Ok, understood, thanks again!

**Comment by nick-harper** [comment]
I have this issue when using Chrome, however when using Firefox it works fine?

**Comment by eisbehr-** [comment]
Without more information I can't help you ...

**Comment by hemberger** [comment]
@eisbehr- I know that it's invalid HTML not to include a `DOCTYPE`, but most things in the browser work fine without one (to the point where you usually can't tell if it has been omitted). I ran into this myself, and concocted all sorts of hypotheses about why it wasn't working before I realized I had forgotten the `DOCTYPE`.

So as to spare forgetful people like myself hours of needless debugging, is there any way to either a) detect this within the js code and emit a warning or b) make some note of it (along with the requirement of image sizes) in the documentation?

Either way, thanks for providing such a useful tool!
